### PR TITLE
Improve date & time formats

### PR DESCRIPTION
**Pull request by sgiehl**
rebased on 3.0 branch.

see #8282 for more details

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This is a changelog for Piwik platform developers. All changes for our HTTP API's, Plugins, Themes, etc will be listed here.
 
+## Piwik 3.0.0
+
+### Breaking Changes
```
Comment (mattab) [inline CHANGELOG.md]: @sgiehl  Can you also note in CHANGELOG that `prettyDate` attribute on API response has slightly changed?

**Comment by mattab** [comment]
@sgiehl could you confirm this nice PR  fixes the issue reported by couple users in the forum http://forum.piwik.org/read.php?2,128505

**Comment by sgiehl** [comment]
@mattab Yes, that fixes those date issues. But that should already be fixed in latest beta of 2.x

**Comment by mattab** [comment]
@sgiehl this PR was merged in 3.0.0 branch - was the issue reported in forum fixed in another PR?

**Comment by sgiehl** [comment]
that was fixed in fb530a11822949e97e9cd0597f667fc2e5f3cf68

File: CHANGELOG.md (hunk 1 above)
Comment (sgiehl) [inline CHANGELOG.md]: done.

**Pull request merged**